internal/storage/bucket_handle.go

```diff
@@ -41,8 +41,12 @@ func (bh *bucketHandle) NewReader(
 	ctx context.Context,
 	req *gcs.ReadObjectRequest) (rc io.ReadCloser, err error) {
 	// Initialising the starting offset and the length to be read by the reader.
-	start := int64((*req.Range).Start)
-	end := int64((*req.Range).Limit)
+	start := int64(0)
```

Comment:
do we need to make it int64 explicitly?

Reply:
yes because req.range is returning in uint64 but in reader method the parameter type is int64.